### DISCUSSION TITLE
Welcome to asustor-platform-driver discussions!

### DISCUSSION THREAD

**Discussion by mafredri** [mafredri/asustor-platform-driver#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GS450** [comment]
Hi. I am just a NAS user with very basic Linux user skills. This is the first time I am participating in this community. I am grateful for the work being done on the Asustor platform drivers. I would like to know if you could kindly consider including driver support for my Asustor model no: AS7010T. I am currently running the Unraid OS on this NAS. Thanks.
